draft/spec/index.html

```diff
@@ -539,12 +539,9 @@ <h2>Manifest</h2>
             <h2>Versions</h2>
             <p>
                 An OCFL Object Inventory MUST include a block for storing versions. This block MUST have the
-                key of <code>versions</code> within the inventory, and it MUST be an array. There is no requirement
-                that the ordering of versions in this array represent a chronological or order of any kind.
```

Comment:
This update drops the [clarification of ordering](https://github.com/OCFL/spec/issues/218).

Reply:
As JSON objects are explicitly unordered, I don't know that we need that?

>  An object is an unordered collection of zero or more name/value
>  pairs, where a name is a string and a value is a string, number,
>  boolean, null, object, or array.

https://tools.ietf.org/html/rfc7159

Reply:
I wondered whether we need any clarification of ordering if we have an object with version keys. I just took a look at the [JSON spec](https://tools.ietf.org/html/rfc7159#section-4) and it leaves the door open for the significance of order.  So, I think that means we should keep this sentence (modulo s/array/object/) but perhaps move it to the top of the [inventory section](https://ocfl.io/draft/spec/#inventory) as a general statement that applies to all objects.

Reply:
The crack in the door I see in RFC7159 is:

> JSON parsing libraries have been observed to differ as to whether or
   not they make the ordering of object members visible to calling
   software.

Reply:
Maybe second sentence in 3.5 intro which is expanded to cover the arrays we use (in which order is also not significant):

> The order of entries in both the [[JSON]] objects and arrays used in inventory files has no significance.

Reply:
I agree with adding a note to section 3.5 that @zimeon suggests two comments up.